src/source/geojson_source.js

```diff
@@ -226,8 +226,8 @@ class GeoJSONSource extends Evented implements Source {
      * For clustered sources, fetches the original points that belong to the cluster (as an array of GeoJSON features).
      *
      * @param clusterId The value of the cluster's `cluster_id` property.
-     * @param limit The maximum number of features to return.
-     * @param offset The number of features to skip (e.g. for pagination).
+     * @param limit The maximum number of features to return. (Defaults to `10` if a falsy value is given.)
```

Comment:
For the record, limit and offset defaults defined in supercluster: https://github.com/mapbox/supercluster/blob/41c28833fe79d53bc8882a4b9db866cefee795c0/index.js#L120